docs/controls/src/imageviewer-all/main.qml

```diff
@@ -0,0 +1,49 @@
+import QtQuick 2.0
+import QtQuick.Controls 2.4
+import QtQuick.Layouts 1.0
+
+ImageViewerWindow {
+    
+    // ...
+
+    id: window
+    
+    width: 640
+    height: 480
+    
+    menuBar: MenuBar {
+        Menu {
+            title: qsTr("&File")
+            MenuItem {
+                text: qsTr("&Open...")
+                icon.name: "document-open"
+                onTriggered: window.openFileDialog()
+            }
+        }
+
+        Menu {
+            title: qsTr("&Help")
+            MenuItem {
+                text: qsTr("&About...")
+                onTriggered: window.openAboutDialog()
+            }
+        }
+    }
+
+    header: ToolBar {
+        RowLayout {
+            anchors.fill: parent
+            ToolButton {
+                text: qsTr("Open")
+                icon.name: "document-open" // https://specifications.freedesktop.org/icon-naming-spec/icon-naming-spec-latest.html
```

Comment:
I would move this comment above the binding since it's quite long.

Reply:
The link is in the text - I removed it from the code.